src/components/LeafletMap.js

```diff
@@ -62,7 +57,7 @@ class LeafletMap extends Component {
           'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
         subdomains: 'abcd',
         minZoom: 0,
-        maxZoom: 10,
+        maxZoom: 11,
```

Comment:
Why are you allowing individuals to zoom deeper than what the tile provider allows to render?